package.json

```diff
@@ -56,6 +57,7 @@
     "@types/node": "16.11.19",
     "husky": "7.0.4",
     "lint-staged": "11.2.6",
+    "outbound-http-logger": "^1.0.3",
```

Comment:
What is this?

Reply:
Something I used for debugging and forgot to remove.  I will be sure to review the diff before requesting PR in the future.